migration-helper/modelmesh-to-raw.sh

```
@@ -0,0 +1,2023 @@
+                # Check if this secret matches the current storage key
+                if [[ -n "$current_storage_key" && "$secret_name" == "$current_storage_key" ]]; then
+                    prioritized_secret="$secret_name"
+                fi
+            fi
+        done <<< "$user_secrets"
+
+        # If no storage key match found but we have a storage URI, check for URI field matches
+        if [[ -z "$prioritized_secret" && -n "$current_storage_uri" ]]; then
+            for secret_name in "${temp_secrets[@]}"; do
+                # Get the secret and check if it has a URI field
+                local secret_data=$(oc get secret "$secret_name" -n "$FROM_NS" -o jsonpath='{.data.URI}' 2>/dev/null || echo "")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)